backend/app/tests/api/routes/test_prompts.py

```
@@ -0,0 +1,277 @@
+    client: TestClient,
+    db: Session,
+    user_api_key: APIKeyPublic,
+):
+    """Test successfully retrieving prompts with pagination metadata"""
+    project_id = user_api_key.project_id
+
+    create_test_prompt(db, project_id, name="prompt_1")
+    create_test_prompt(db, project_id, name="prompt_2")
+
+    response = client.get(
+        f"/api/v1/prompts/?skip=0&limit=100",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)